src/Logging/Internal/EventSource/EmptyServiceContext.cs

```diff
@@ -8,10 +8,16 @@ namespace Microsoft.Omex.Extensions.Logging
 	/// <summary>
 	/// IServiceContext without any information
 	/// </summary>
-	internal class EmptyServiceContext : IServiceContext
+	public class EmptyServiceContext : IServiceContext
```

Comment:
Why make this public?

Reply:
Because we need to register this in Logging/ServiceCollectionExtensions.cs which is in a different project.

Reply:
So in summary, the change over to Open Telemetry is resulting in this the solution in this repo becoming incomplete.
If that is the case then why keep this repo alive at all and why not just move it to the internal repo that completes the solution?

Reply:
Logging/ServiceCollectionExtensions.cs is not in a different project.

Reply:
> Logging/ServiceCollectionExtensions.cs is not in a different project.

I mean making EmptyServiceContext public so that we can register EmptyServiceContext in `Hosting.Services\HostBuilderExtensions.cs`

Reply:
> So in summary, the change over to Open Telemetry is resulting in this the solution in this repo becoming incomplete. If that is the case then why keep this repo alive at all and why not just move it to the internal repo that completes the solution?

In future, we want to have another OmexOpenTelemetryLogger opensource as discussed with the team

Reply:
yes